### PR TITLE
AC-6902: Fix failing impact-api build

### PR DESCRIPTION
**Pull request by VincentHokie**
#### Changes introduced in [AC-6902](https://masschallenge.atlassian.net/browse/AC-6902)
- update the web/Dockerfile and web/Dockerfile.travis files
  to install the mariadb-client in place of the mysql-client
- add the graphql-core to the web/impact/requirements/base.txt file
  so that it's version and that of graphene match

#### How to test
- a passing travis should be good for this

### CODE REVIEW

**Review by shanbady — APPROVED** [review]
seems to have fixed the issue on my end and on travis.

File: web/Dockerfile [hunk 1]
```diff
@@ -8,9 +8,10 @@ COPY scripts/start-nodaemon.sh /usr/bin
 COPY scripts/mysqlwait.sh /usr/bin
 
 # python2.7 is required to run supervisor
-RUN apt-get update -y && apt-get install -y \
+RUN apt-get update -y && apt-get upgrade && \
```
Comment (shanbady) [inline web/Dockerfile]: @VincentHokie  im getting a prompt when running the 'upgrade' step. does it need the "-y" flag?

**Review by shanbady — CHANGES_REQUESTED** [review]

File: web/Dockerfile (hunk 1 above)
Comment (VincentHokie) [reply web/Dockerfile]: ah, must have missed this..adding

Comment (VincentHokie) [reply web/Dockerfile]: weird, it doesn't prompt me..won't hurt to make the command more complete though

**Review by kaymation — APPROVED** [review]
Looks straightforward to me. my thumb is contingent on shanks

**Review by shanbady — APPROVED** [review]

**Comment by shanbady** [comment]
will merge after build passes

**Pull request merged**